--- bs-2.5/Makefile.orig	2003-12-25 11:28:35.000000000 +0100
+++ bs-2.5/Makefile	2003-12-25 11:31:07.000000000 +0100
@@ -6,6 +6,9 @@
 TERMLIB = -lncurses
 CC = gcc
 
+# DESTDIR support
+DESTDIR = 
+
 bs: bs.c
 	$(CC) $(CFLAGS) -o bs bs.c $(TERMLIB)
 
@@ -16,8 +19,8 @@
 	rm -f bs bs.6 bs*.tar.gz bs*.rpm *~
 
 install: bs.6 uninstall
-	cp bs /usr/bin
-	cp bs.6 /usr/share/man/man6/bs.6
+	cp bs $(DESTDIR)/usr/bin
+	cp bs.6 $(DESTDIR)/usr/share/man/man6/bs.6
 
 uninstall:
 	rm -f /usr/bin/bs /usr/share/man/man6/bs.6
