--- bs-2.13/Makefile.orig	2024-12-01 20:14:11.628052404 +0100
+++ bs-2.13/Makefile	2024-12-01 20:36:48.880699535 +0100
@@ -11,7 +11,7 @@ CC = gcc
 DESTDIR = 
 
 bs: bs.c
-	$(CC) $(CFLAGS) -DVERSION='"$(VERSION)"' -o bs bs.c -lncurses
+	$(CC) $(CFLAGS) $(LDFLAGS) -DVERSION='"$(VERSION)"' -o bs bs.c $(TERMLIB)
 
 # Note: to suppress the footers with timestamps being generated in HTML,
 # we use "-a nofooter".
